laserstudio/__init__.py

```diff
@@ -4,5 +4,5 @@
 Hardware evaluation tool developed by the Donjon
 """
 
-__version__ = "1.0.0dev3"
+__version__ = "0.0.0"
```

Comment:
Was expecting "1.0.0", is this correct?

Reply:
It is something automatically updated by poetry when publishing with the git tag version as "v1.0.0". Bumping will be done in main branch